Modules/Bridge/NumPy/wrapping/PyBuffer.i.init

```diff
@@ -2,7 +2,20 @@
 
 HAVE_NUMPY = True
 try:
-  import numpy
+  import numpy as np
+  class NDArrayITKBase(np.ndarray):
+      """A numpy array with an optional itk object "base" object to used to
```

Comment:
clean up sentence

Reply:
Good catch.

Much improved now: *A numpy array that provides a view on the data associated with an option itk "base" object.*